### DISCUSSION TITLE
OSU Lazer doesn't want to create my account and I am ver ANGY 😡😡😡😡

### DISCUSSION THREAD

**Discussion by AnnoxianMC** [ppy/osu#23343] [Q&A]
So I downloaded OSU Lazer for playing it but, when I was going to create an account, it showed me this error: Something happened... we are not sure what. If someone can help me or a OSU developer.
![IMG_20230430_114923](https://user-images.githubusercontent.com/120185991/235346789-0b1fbda0-4281-45e7-9e7b-77422ceb6b1a.jpg)

**Comment by zzahkaboom24** [comment]
Create your account on the osu! website, not within osu!lazer.

This is the osu! website -> https://osu.ppy.sh/

**Reply by bdach** [reply]
pray tell, how do you propose the OP does that? hint:

![1682857722](https://user-images.githubusercontent.com/20418176/235353031-f266ceda-8bcf-4859-9e85-50c2659ba1d9.png)

the website says _you need the client to create an account_. please don't answer discussions if you don't know what you're talking about.

**Comment by bdach** [comment]
@AnnoxianMC please upload logs after trying to create the account, as requested in https://github.com/ppy/osu/discussions/15188

**Comment by MineTurtlee** [comment]
Please upload logs here so we can tell what it is :|
